### PR TITLE
Adds dependabot.yml

### PR DESCRIPTION
**Pull request by GreenGard**
Add dependabot support for Maven, Github actions and Docker. Solves issue #10

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  # Maintain dependencies for Maven
+  - package-ecosystem: "maven"
+    directory: "/.github/workflows/"
+    schedule:
+      interval: "daily"
+
+    # Maintain dependencies for Docker
+  - package-ecosystem: "docker"
+    directory: "/.github/workflows/"
```
Comment (kappsegla) [inline .github/dependabot.yml:17]:
```suggestion
    directory: "/
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+  # Maintain dependencies for Maven
+  - package-ecosystem: "maven"
+    directory: "/.github/workflows/"
```
Comment (kappsegla) [inline .github/dependabot.yml:11]:
```suggestion
    directory: "/"
```

**Review by kappsegla — CHANGES_REQUESTED** [review]
Shouldn't the directory be / for all checks?

**Review by kappsegla — APPROVED** [review]
As far as I understand this will work as intended.

**Comment by HelenaHalldinITHS** [comment]
Looks good!

**Review by HelenaHalldinITHS — APPROVED** [review]
Looks good!

**Pull request merged**